src/lib/rule-finder.js

```diff
@@ -84,13 +84,22 @@ function _getAllAvailableRules(pluginRules) {
   return filteredAllRules
 }
 
-function RuleFinder(specifiedFile) {
+function _isNotCore(rule) {
+  return rule.indexOf('/') !== '-1'
+}
+
+function RuleFinder(specifiedFile, noCore) {
   var configFile = _getConfigFile(specifiedFile)
   var config = _getConfig(configFile)
   var currentRules = _getCurrentRules(config)
   var pluginRules = _getPluginRules(config)
-  var allRules = _getAllAvailableRules(pluginRules)
-  var unusedRules = difference(allRules, currentRules)
+  var allRules = pluginRules
+  if (noCore) {
+    currentRules = currentRules.filter(_isNotCore)
+  } else {
+    allRules = _getAllAvailableRules(pluginRules)
+  }
+  var unusedRules = difference(allRules, currentRules) // eslint-disable-line vars-on-top
 
```

Comment:
Out of curiosity, did you have a particular reason to filter the rules the way you did in the above lines vs. within the methods below (controlled by a corresponding parameter)?